sample/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@types/jest": "^24.9.0",
     "firebase": "^7.9.1",
+    "js-levenshtein": "^1.1.6",
```

Comment:
why is this needed?

Reply:
TBH, not sure, new peer dep somewhere in the tree. The demo app wouldn't compile without it.

Reply:
```
Failed to compile.

../node_modules/react-dev-utils/webpackHotDevClient.js
Error: [BABEL] /Users/james/Code/Firebase/reactfire/node_modules/react-dev-utils/webpackHotDevClient.js: Cannot find module 'js-levenshtein' (While processing: "/Users/james/Code/Firebase/reactfire/node_modules/babel-preset-react-app/dependencies.js")
```

Reply:
Huh. I bumped the react version on master - maybe you need a fresh `yarn install`?